backend/controllers/moviesController.js

```diff
@@ -0,0 +1,44 @@
+
+
+// const services = 'netflix, prime.subscription';
+// const country = 'us';
+// const genres = '878';
+// const genres_relation = 'or';
+
+// const url = `https://streaming-availability.p.rapidapi.com/search/filters?services=netflix,prime.subscription&country=us&genres=10749,35&genres_relation=or`;
+
+
+const options = {
+  method: 'GET',
+  headers: {
+    'X-RapidAPI-Key': 'f267b612b6msh504ab02ca095331p142270jsn257fad83de1e',
+    'X-RapidAPI-Host': 'streaming-availability.p.rapidapi.com'
+  }
+};
+
+const MoviesController = {
```

Comment:
i believe controller name should be lower case

Reply:
changed